_data/wallets-collection.yml

```diff
@@ -42,13 +42,13 @@
   links:
     -
       label: Download ZIP
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/dashcore-0.12.2.3-win64.zip"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3.1/dashcore-0.12.3.1-win64.zip"
     -
       label: Hash File
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/SHA256SUMS.asc"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3.1/SHA256SUMS.asc"
     -
       label: PGP
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/dashcore-0.12.2.3-win64.zip.asc"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3/dashcore-0.12.3-win64.zip.asc"
```

Comment:
There is an inconsistency - all other `PGP`s are disabled. Also, there is no such file.